plejd/README.md

```diff
@@ -1,9 +1,32 @@
 # Hass.io Plejd add-on
 
 Hass.io add-on for Plejd home automation devices. Gives you the ability to control the Plejd home automation devices through Home Assistant.
-It uses MQTT to communicate with Home Assistant and supports auto discovery of the devices in range.
-
-It also supports notifications so that changed made in the Plejd app are propagated to Home Assistant.
+It uses MQTT to communicate with Home Assistant and supports auto discovery of the devices in range. Changed made in the Plejd app are propagated to Home Assistant.
+
+Plejd output devices typically appears as either lights or switches in Home Assistant depending on how they are configured.
+<table>
```

Comment:
I find this a bit unreadable. Can we use the | | markdown syntax?

```
| Device | Plejd Configuration | Home Assistant Role | Comment                                                               |
|--------|---------------------|---------------------|-----------------------------------------------------------------------|
| CTR-01 | Relay, Light        | Light               |                                                                       |
| CTR-01 | Relay, Other        | Switch              |                                                                       |
| REL-01 | Relay, Light        | Light               |                                                                       |
| REL-01 | Relay, Other        | Switch              |                                                                       |
| REL-02 | Relay, Light        | Light               |                                                                       |
| REL-02 | Relay, Other        | Switch              |                                                                       |
| SPR-01 | Relay, Light        | Light               | Not tested, not released by Plejd                                     |
| SPR-01 | Relay, Other        | Switch              | Not tested, not released by Plejd                                     |
| DIM-01 | -                   | Light               |                                                                       |
| DIM-02 | -                   | Light               |                                                                       |
| LED-10 | -                   | Light               |                                                                       |
| DAL-01 | -                   | -                   | Not supported, not released by Plejd                                  |
| WPH-01 | -                   | Device Automation   | type:button_short_press, subtype:button_1, button_2,button_3,button_4 |
| WRT-01 | -                   | Device Automation   | type:button_short_press, subtype:button_1                             |
| GWY-01 | -                   | -                   |                                                                       |
| RTR-01 | -                   | -                   |                                                                       |
| Scene  | -                   | Scene               |                                                                       |
| Scene  | -                   | Device Automation   | type:scene, subtype:trigger                                           |
| Room   | -                   | Area                | Can be changed by Home Assistant                                      |
| Room   | -                   | Light               | If includeRoomsAsLights is set to true                                |
```

Reply:
Another thing - maybe "not released" is a bad phrasing since it might not be true for the duration of the release. Maybe just mark them as "not tested, might work" / "not supported" or similar?